slides/ds101-worskhop.mdx

```diff
@@ -127,12 +127,21 @@ We had a lot of fun!
 
 <Primary imgSrc="../assets/toast2.jpg">
 
-# Recap
+# Review
 
 </Primary>
 
 ---
 
+## What we learned making toasts
+
+- Got to know each other
+- Worked in Zoom rooms
```

Comment:
Let’s not use past sense here

Reply:
@okonet I fixed wording in this and one more piece.